docs: add contributing and template (#22)
catppuccin-rfc/userstyles: README.md

<h3 align="center">
  <img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/logos/exports/1544x1544_circle.png" width="100" alt="Logo"/><br/>
  <img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/misc/transparent.png" height="30" width="0px"/>
  Catppuccin Userstyles
  <img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/misc/transparent.png" height="30" width="0px"/>
</h3>

<h6 align="center">

>   <a href="#%EF%B8%8F-install">Install</a>

Reviewer: I think you can just change it to this:
```suggestion
  <a href="#-install">Install</a>
```

  .
  <a href="#-contributing">Contributing</a>
  ·
  <a href="#-userstyles">Userstyles</a>
</h6>

&nbsp;

<p align="center">

> Catppuccin userstyles is a collection of userstyles for various websites and web applications that are themed with the soothing pastel colors that we all love.

Reviewer: We should try to aim this description more like the main repository and pad it out a bit more.

1. What is a userstyle? A short one line sentence would be nice.
2. We should add the https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/palette/macchiato.png and also include the shields for star count, issues etc

Author: I created the sentence:
"A userstyle is a theme that applies to a given website with alterations to the css of the site."
However, this maybe a little complex.

&nbsp;

<p align="center">

> Catppuccin userstyles is a collection of userstyles for various websites and web applications that are themed with the smoothing pastel colors that we all love.

Reviewer 2: ```suggestion
Catppuccin userstyles is a collection of userstyles for various websites and web applications that are themed with the soothing pastel colors that we all love.
```

    - Replacing `<name>` with the name of the userstyle.
4. Enjoy!

> **Note** <br>

> > If you are using Oprea please see [oprea.md](docs\oprea.md) for instructions as it differs from the normal process.

Reviewer 2: Opera stylus is deprecated.

Author: Opera stylus maybe deprecated. However, since the engine is now using chromium it can run therefore I just wanted to add this in since it was a little niche.

Reviewer 3: This should be renamed to Opera / `opera.md`, if you're going to keep this.

&nbsp;
### 👐 Contributing

> **Note** <br>

> > if you would like to submit a port or learn how to port Catppuccin to a program, please refer to the [port-creation.md](docs/port-creation.md) file.

Reviewer 2: ```suggestion
> if you would like to submit a userstyle or learn how to port Catppuccin to a website, please refer to the [port-creation.md](docs/port-creation.md) file.
```
